test/jump_wrapper.jl

```diff
@@ -0,0 +1,124 @@
+# Copyright (c) 2020: Akshay Sharma and contributors
+#
+# Use of this source code is governed by an MIT-style license that can be found
+# in the LICENSE.md file or at https://opensource.org/licenses/MIT.
+
+module TestJuMPWrapper
+
+using Test
+using JuMP
+import DiffOpt
+import HiGHS
+import Ipopt
+import SCS
+import MathOptInterface as MOI
+
+const ATOL = 1e-3
+const RTOL = 1e-3
+
+function runtests()
+    for name in names(@__MODULE__; all = true)
+        if startswith("$name", "test_")
+            @testset "$(name)" begin
+                getfield(@__MODULE__, name)()
+            end
+        end
+    end
+    return
+end
+
+function test_jump_api()
+    for (MODEL, SOLVER) in [
+            (DiffOpt.diff_model, Ipopt.Optimizer),
+            (DiffOpt.quadratic_diff_model, HiGHS.Optimizer),
+            (DiffOpt.quadratic_diff_model, SCS.Optimizer),
+            (DiffOpt.quadratic_diff_model, Ipopt.Optimizer),
+            # (DiffOpt.conic_diff_model, HiGHS.Optimizer),
+            # (DiffOpt.conic_diff_model, SCS.Optimizer), # conicmodel has a issue with sign
```

Comment:
We need at least one of the three to be enabled otherwise it's not tested

Reply:
Yup,
But there is a bug.
Conic returns wrong sign in some cases.